### ISSUE TITLE
ResampleStep fails when --output_dir is used in calwebb_image3

### ISSUE THREAD

**Issue by stscieisenhamer** [spacetelescope/jwst#2004]
# Issue

When `--output_dir` is specified for `calwebb_image3`, the resample step fails because it is using `blendheaders` on the `crf` files, but without adding the output directory path.

Consider using `blendmodels` instead and do the blending in-memory.

**Comment by stscieisenhamer** [comment]
# Resources
A test dataset can be found in
```
/grp/jwst/ssb/eisenhamer/test_wfss/sdp_20180412/level2b
```
Use the command:
```
strun ../cfgs/calwebb_image3.cfg jw87800-o017_image3_001_asn.json --output_dir=/tmp
```
or set `output_dir` to wherever is good.

**Comment by jdavies-st** [comment]
This has been fixed, likely in #2024.
